l10n_it_split_payment/README.rst

```diff
@@ -15,18 +15,26 @@ Configuration
 
 To configure this module, you need to:
 
-* go to Settings, Configuration, Accounting and configure 'Split Payment Write-off account' (like 'IVA n/debito sospesa SP') and 'Split Payment Write-off Journal'. Write-off account should be different from standard debit VAT, in order to separately add it in VAT statement.
+* go to Settings, Configuration, Accounting and configure 'Split Payment Write-off account' (like 'IVA n/debito sospesa SP'). Write-off account should be different from standard debit VAT, in order to separately add it in VAT statement.
 * configure the fiscal position used for split payment, setting 'Split Payment' flag. In fiscal position, map standard VAT with SP VAT, like the following:
```

Comment:
please specify the right menu: Invoice, Configuration, Accounting, Fiscal Positions